src/generators/legacy.js

```diff
@@ -12,10 +12,10 @@ module.exports = function (that, compiler) {
             content: that.options['theme-color'] || that.options.theme_color
           }
           if (themeColorTag.content) applyTag(tags, 'meta', themeColorTag)
-          applyTag(tags, 'link', {
+          applyTag(tags, 'link', Object.assign({
             rel: 'manifest',
             href: that.manifest.url
-          })
+          }, !!that.options.crossorigin && { crossorigin: that.options.crossorigin }))
```

Comment:
should this handle `crossOrigin` case-insensitively?

Reply:
I think we should just keep crossorigin because crossOrigin is the property and crossorigin is the actual attribute that gets added to the html. This keeps it consistent with the other options which are also case-sensitive.

Reply:
sounds good to me :+1:

Reply:
sounds good. can you do `that.options.crossorigin ? { crossorigin: that.options.crossorigin } : {}`?